src/script/main/app.ts

```diff
@@ -413,10 +414,6 @@ export class App {
         conversationRepository.initMLSConversationRecoveredListener();
       }
 
-      if (connections.length) {
-        await Promise.allSettled(conversationRepository.mapConnections(connections));
-      }
-
```

Comment:
Correct, this logic was moved to `init1To1Conversations`, I don't quite get what you mean, you've commented on removed code 🤔

Reply:
My question is: would it make sense to have a separate function that does the connection mapping (rather than doing it inside of the `init1to1conversations` method). In order to split reponsibilities. But maybe it doesn't make sense, I leave you judge of this :D